vignettes/queries_dataset.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+title: "Queries dataset documentation"
+output:
+  rmarkdown::html_vignette:
+  toc: false
+vignette: >
+  %\VignetteIndexEntry{Queries dataset documentation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+  ```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
```

Comment:
There's spaces before the backticks which might cause a warning when knitting.

Reply:
updated